data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java

```diff
@@ -103,12 +104,13 @@ private static void jettyEnableInherit(Server server) {
     @Bean
     public CommandLineRunner loadData(DataInitializer initializer) {
         return (args) -> {
-            initializer.initialize();
+            // initializer.initialize();
```

Comment:
We need to discuss a way to start the data server with fake data for local UI development where real data is not needed. This is closely related to the implementation of a persistent database.
If we decide to completely get rid of the fake data we should also remove the associated code and infrastructure.